### ISSUE TITLE
migration issue

### ISSUE THREAD

**Issue by richard-allen** [hodfords/zimbra-migration#1]
Hello.

I'm using your scripts to migrate my Zimbra 8.8.15 OSS server on CentOS 7 to 10.1.3 on RHEL 9.  The software Im using is from here:

https://maldua.github.io/zimbra-foss-builder/downloads.html

Everything seems to be in order after the migration but now the server seems to count all the galsync users that got migrated over as normal users.  The old 8.8.15 server tells me it has 20 accounts and does not display any galsync users.
The 10.1.3 server has 44 accounts and lists galsync users, spam, ham and virus  users as well.

Im wondering if I could have real (hidden) galsync and spam users and then also the ones that migrated over.

Or is this just norm for the new version? :)

PS: thanks for a great script.

**Comment by richard-allen** [comment]
I also just noticed that domain aliases got migrated over as real domains.
The new domains that got created that used to be domain aliases do not have a visible galsync user

**Comment by hodfords** [comment]
Sorry for the last reply... thanks for this...  

Are you using the scripts hereunder for the migration or are you using the script that you have referenced? If the latter we wouldn't be able to support. 

(1) For all of the galsync@ and spam@ email accounts - what are you expecting? Should we omit them in the export script? or omit them from the import script 
(2) Domain name aliases - I didn't realise it was possible to create aliases for domain names.

**Comment by richard-allen** [comment]
Yes I used your migration scripts.   The link I provided is not for a migration system between zimbra servers, but a complete rebuild of the zimbra server software.
I don't know how the galsync users are supposed to work, but they seem to be created when a domain is created and do not seem count towards the usercount for licenses.  I THINK your script copies the galsync users from the source-server and creates them as regular users thus raising the count of users once for every domain added.
But I think the expectation has to be the same number of users counted/shown on the target server.
But again, Im not sure how it works.

Domain aliases do exist and are a very cool feature.  You can have multiple domains all pointing to the same domain and users.   I'll attach a screenshot

**Comment by richard-allen** [comment]
![Screenshot From 2025-01-02 11-18-56](https://github.com/user-attachments/assets/e3f46a61-e868-428e-8dc8-7de49504993b)
